cmake/modules/FindODBC.cmake

```diff
@@ -17,8 +17,11 @@
 # ODBC_LIBRARY, where to find the ODBC driver manager library.
 
 set(ODBC_FOUND FALSE)
+include(CheckIncludeFiles)
+check_include_files("windows.h;sqlext.h" HAVE_SQLEXT_H)
 
-find_path(ODBC_INCLUDE_DIR sql.h
+find_path(ODBC_INCLUDE_DIR sqlext.h
+  /include
```

Comment:
Just curious, do any platforms/compilers really have any headers in `/include`? I think I've never encountered this...